js-miniapp-bridge/src/common-bridge.ts

```diff
@@ -263,6 +264,20 @@ export class MiniAppBridge {
     });
   }
 
+  /**
+   * Get access token from native hostapp.
+   */
+  getAccessToken() {
+    return new Promise<AccessTokenData>((resolve, reject) => {
+      return this.executor.exec(
+        'getAccessToken',
+        null,
+        tokenData => resolve(JSON.parse(tokenData) as AccessTokenData),
```

Comment:
I am not sure, if only `resolve(JSON.parse(tokenData))` here is enough?

Reply:
I have updated. Define the type here help on clearing the context 😄.